src/controllers/common.go

```diff
@@ -1,6 +1,42 @@
 package controllers
 
-import "github.com/dana-team/platform-backend/src/types"
+import (
+	"fmt"
+	"github.com/dana-team/platform-backend/src/types"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"regexp"
+	"strings"
+)
+
+// checkLabelSelectorFormat checks if the input string is in the format "a=b,c=d,e=f".
+func checkLabelSelectorFormat(labelSelector string) bool {
+	pattern := `^([^\s=,]+)=([^\s=,]+)(,([^\s=,]+)=([^\s=,]+))*$`
```

Comment:
What will happen if the value is:

1. `a!=b`
2. `a="b"`
3. `a=(b)`
4. `+=b!!!`

Can we check edge cases? or it's better to use native Kubernetes's labels package, where you have similar functionality that checks all edge cases [1]:
```
// Parse takes a string representing a selector and returns a selector object, or an error.
func Parse(selector string, opts ...field.PathOption) (Selector, error)
```
[1] https://pkg.go.dev/k8s.io/apimachinery/pkg/labels#Parse

Reply:
Good call, this is better